src/utils/headers.js

```diff
@@ -1,64 +1,22 @@
-const escapeRegExp = require('lodash/escapeRegExp')
-const trimEnd = require('lodash/trimEnd')
 const { parseAllHeaders } = require('netlify-headers-parser')
 
 const { NETLIFYDEVWARN } = require('./logo')
 
-// Our production logic uses regex too
```

Comment:
This whole function is now exposed by `netlify-headers-parser` as a `forRegExp` property.